betty/extension/cotton_candy/__init__.py

```diff
@@ -175,15 +173,25 @@ def dump(self) -> VoidableDump:
         })
 
 
-class _CottonCandy(Theme, ConfigurableExtension[CottonCandyConfiguration], Generator, GuiBuilder, _NpmBuilder, Jinja2Provider):
+class _CottonCandy(Theme, CssProvider, ConfigurableExtension[CottonCandyConfiguration], GuiBuilder, Jinja2Provider, _WebpackEntrypointProvider):
     @classmethod
     def depends_on(cls) -> set[type[Extension]]:
-        return {_Npm}
+        return {_Webpack}
+
+    @classmethod
+    def comes_after(cls) -> set[type[Extension]]:
+        return {_Maps, _Trees}
 
     @classmethod
     def assets_directory_path(cls) -> Path | None:
         return Path(__file__).parent / 'assets'
 
+    @property
+    def public_css_paths(self) -> list[str]:
```

Comment:
This must instead be added in base.html.j2. Right now, this would be added even for pages without Cotton Candy.